docs/index.rst

```diff
@@ -70,6 +70,7 @@ If you would like to contribute to SecureDrop, please see our
    what_makes_securedrop_unique
    logging
    ossec_alerts
+   remote_svs
```

Comment:
Could we cross-link this guide from any other relevant spots, e.g., the SVS setup guide?